index.html

```diff
@@ -1,407 +1,433 @@
+          LET’S CONNECT
+        </h4>
+        <ul class="flex-list">
+          <li>
+            <a href="https://twitter.com/christopheruduekwe">
+              <img src="assets/images/social/bird.png" 
+                class="img-social" alt="angellist icon">
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/christopher-uduekwe/">
+              <img src="assets/images/social/Linkedin icon.png" 
+                class="img-social" alt="Linkedin icon">
+            </a>
+          </li>
+          <li>
+            <a href="http://">
+              <img src="assets/images/social/medium.png" 
+                class="img-social" alt="medium icon">
+            </a>
+          </li>
+          <li>
+            <a href="http://github.com/chudisoft">
+              <img src="assets/images/social/github.png" 
+                class="img-social" alt="github icon">
+            </a>
+          </li>
+          <li>
+            <a href="/">
+              <img src="assets/images/social/angellist.png" 
+                class="img-social" alt="angellist icon">
+            </a>
+          </li>
+        </ul>
+        <div>
+          <button class="project-button" type="button"
+            onclick="return;">
+            Get my resume
+          </button>
+        </div>
+        <ul class="tech-stack">
+          <li class="stack-area">
+            <div>
+              <div class="stack-btn">
+                <h5 class="stack-title">Languages</h5>
+                <div>
+                  <img src="assets/images/icons/arrowdown.png" 
+                    alt="dropdown icon" class="stack-img"/>
                 </div>
               </div>
-            </li>
-            <li class="stack-area">
-              <div>
-                <div class="stack-btn">
-                  <h5 class="stack-title">Frameworks</h5>
-                  <img
-                    src="assets/images/icons/arrowright.png"
-                    alt="dropdown icon"
-                    class="stack-img"
-                  />
+              <div class="">
+                <div class="grid-container">
+                  <img src="assets/images/icons/Ellipse 1.png" 
+                    alt="Javascript" class="grid-item-icon"/>
+                  <label class="grid-item-text">JavaScript</label>
+                </div>
+                <div class="grid-container">
+                  <img src="assets/images/icons/Ellipse 1-1.png" 
+                    alt="Javascript" class="grid-item-icon"/>
+                  <label class="grid-item-text">HTML</label>
                 </div>
-                <div class="hidden">
-                  <div class="grid-container"></div>
+                <div class="grid-container">
+                  <img src="assets/images/icons/Ellipse 1-2.png" 
+                    alt="Javascript" class="grid-item-icon"/>
+                  <label class="grid-item-text">CSS</label>
                 </div>
```

Comment:
- [ ] Please add the paragraph as in the given Figma design. Currently, you don't have a paragraph about why people reach out to you. Kindly, implement it.